users.js

```diff
@@ -120,29 +137,35 @@ module.exports = function (options) {
 
       args.mailingList = (args.mailingList) ? 1 : 0;
 
-      seneca.act({role: 'user', cmd: 'register'}, args, function (err, registerResponse) {
+      checkPassword(args, function (err, args) {
         if (err) return done(err);
-        if (!registerResponse.ok) {
-          return done(null, registerResponse);
+        if (typeof args.ok !== 'undefined' && !args.ok) {
```

Comment:
Don't we specifically want to just check if args.ok is false? What should happen when it's null or 0? Should we consider that not ok as well? Since we're in full control of what ok's value is, I'd be in favour of simplifying to
```
if (args.ok === false) {
```